### DISCUSSION TITLE
Do we need a Glossary?

### DISCUSSION THREAD

**Discussion by wildintellect** [cloudnativegeo/cloud-optimized-geospatial-formats-guide#65] [Ideas]
Reading through some pages, there are a lot terms that won't be familiar to most people who don't already know the format well. I think we need a glossary page unless we plan to explain terms inline on the page where they occur (might only be relevant to one format).

Some examples from the Zarr page https://guide.cloudnativegeo.org/zarr/intro.html
* MutableMapping
* Object Storage
* numpy arrays

We also need to be careful to try not to use Python specific terminology outside of coding examples.

**Comment by kylebarron** [comment]
Yeah I think a glossary could make sense. I've been trying to explain terms inline on the page, but I'm sure I miss some terms that should be better described.

**Comment by kcarini** [comment]
I think a glossary for this is important in terms of increasing the accessibility of the material.

**Comment by wildintellect** [comment]
https://guide.cloudnativegeo.org/glossary.html
